[Bug]: ERR_PNPM_TRUST_DOWNGRADE  High-risk trust downgrade for "chokidar@4.0.3" (possible package takeover)

### Reproduction

ERR_PNPM_TRUST_DOWNGRADE  High-risk trust downgrade for "chokidar@4.0.3" (possible package takeover)

### Describe the bug

ERR_PNPM_TRUST_DOWNGRADE  High-risk trust downgrade for "chokidar@4.0.3" (possible package takeover)

### System Info

```bash
ERR_PNPM_TRUST_DOWNGRADE  High-risk trust downgrade for "chokidar@4.0.3" (possible package takeover)
```

### Contributes

- [ ] I am willing to submit a PR to fix this issue
- [ ] I am willing to submit a PR with failing tests

--- a/pnpm-workspace.yaml
+++ b/pnpm-workspace.yaml
@@ -54,3 +54,9 @@ onlyBuiltDependencies:
   - unrs-resolver
   - vue-demi
   - workerd
+
+trustPolicyExclude:
+  - "chokidar@4.0.3"
+  - "koa@2.16.3"
+  - "semver@6.3.1"
+  - "tailwindcss@3.4.18"

